### DISCUSSION TITLE
[Interconnect & Memory] Compiler Infrastructure for Specializing Domain-Specific Memory Templates

### DISCUSSION THREAD

**Discussion by chrpilat** [cucapra/latte21#34] [Papers]
Authors: Stephanie Soldavini and Christian Pilato
Paper PDF: [https://capra.cs.cornell.edu/latte21/paper/9.pdf](https://capra.cs.cornell.edu/latte21/paper/9.pdf)
Talk Recording: [https://youtu.be/PugsMNbdRQE](https://youtu.be/PugsMNbdRQE)

**Comment by stephenneuendorffer** [comment]
Thanks for the talk! I'm curious if you've gotten to the point of having a sketch of the CIRCT dialect?

**Reply by chrpilat** [reply]
Not yet, but we are working on it! I will keep you posted on it if interested.
